### ISSUE TITLE
Allow use for more gpu options

### ISSUE THREAD

**Issue by Mikeonut** [vosen/ZLUDA#67]
Seems great even if it only improves render times in something like blender by a few seconds, but it blocks out the use of Nvidea gpus to work along side amd ones in cuda. That would be a nice feature to have.

**Comment by YandiBanyuKarimaWaly** [comment]
> Seems great even if it only improves render times in something like blender by a few seconds, but it blocks out the use of Nvidea gpus to work along side amd ones in cuda. That would be a nice feature to have.

I don't think that should be implemented yet as there is still a lot to do if this were going to mature. Also, those use cases may be very small since not a lot of people use dual cards on their system, much less of mixed brand-type (I don't have data on this one, just anecdote)

**Comment by vosen** [comment]
If ZLUDA development continues then that will be on the todo list.
For now, ZLUDA is written to make sure that it's "non-invasive" - an application runs ZLUDA only if you specifically inject it. So an app can't use both NV and AMD GPU, but if you have both GPUs then you can at least choose
